src/storage/admin/AdminTask.h

```diff
@@ -33,6 +33,8 @@ class AdminSubTask {
 
 enum class TaskPriority : int8_t { LO, MID, HI };
 
+enum class AdminTaskStatus : int8_t { QUEUE, RUNNING, FINISHED, CANCELED };
```

Comment:
'isRunning()' is enough, there is no need to introduce an new status, it will caused confused with the old one.